Cargo.toml

```diff
@@ -48,6 +48,7 @@ tokio = { version = "1", features = [
     "io-util",
     "fs",
     "net",
+    "rt-multi-thread"
```

Comment:
No hard blocker for me, just out of curiosity: why do we need the parallel runtime for the CLI?

Reply:
My guess is that they changed the feature defaults in tokio 1.37.0. At least I saw this error after running `cargo update`:
```
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
   --> src/lib.rs:255:13
    |
255 |             #[tokio::main]
    |             ^^^^^^^^^^^^^^
    |
    = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)
```